modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/loading/index.js

```diff
@@ -15,6 +15,11 @@
 
 const baseURL = Liferay.ThemeDisplay.getPortalURL();
 const myUserId = Liferay.ThemeDisplay.getUserId();
+const accountGroups = {
+	'': 'home',
```

Comment:
@juliocezar96 is not common to add an empty object entry as value, can you remove?

```suggestion
```